python/doc/examples/functional_modeling/logistic_growth_model.ipynb

```diff
@@ -0,0 +1,305 @@
+    "- $t_0$ is the initial time,\n",
+    "- $y_0$ is the initial population at time $t=t_0$, \n",
+    "- $t_{final}$ is the final time.\n",
+    "\n",
+    "The $a$ parameter sets the growth rate of the population. The $b$ parameter acts as a competition parameter which limits the size of the population by increasing the competition between its members. \n",
+    "\n",
+    "In [1], the author uses this model to simulate the growth of the U.S. population. To do this, the author uses the U.S. census data from 1790 to 1910. For this time interval, R. Pearl and L. Reed [2] computed the following values of the parameters:\n",
+    "\n",
+    "$$\n",
+    "a = 0.03134, \\qquad \n",
+    "b = 1.5887 \\times 10^{-10}.\n",
+    "$$\n",
+    "\n",
+    "Our goal is to use the logistic growth model in order to simulate the solution for a larger time interval, from 1790 to 2000:\n",
+    "\n",
+    "$$\n",
+    "t_0 = 1790, \\qquad \n",
+    "t_{final} = 2000.\n",
+    "$$\n",
+    "\n",
+    "Then we can compare the predictions of this model with the real evolution of the U.S. population.\n",
+    "\n",
+    "We can prove that, if $y_0 > 0$, then the limit population is:\n",
+    "\n",
+    "$$\n",
+    "y_{limit} =\\frac{a}{b}.\n",
+    "$$\n",
+    "\n",
+    "In 1790, the U.S. population was 3.9 Millions inhabitants:\n",
+    "\n",
+    "$$\n",
+    "y_0 = 3.9 \\times 10^6.\n",
+    "$$\n",
+    "\n",
+    "We can prove that the exact solution of the ordinary differential equation is:\n",
+    "\n",
+    "$$\n",
+    "y(t)=\\frac{ay_0}{by_0+(a-by_0 ) \\exp(-a(t-t_0)) }\n",
+    "$$\n",
+    "\n",
+    "for any $t\\in[t_0, t_{final}]$.\n",
+    "\n",
+    "We want to see the solution of the ordinary differential equation when uncertainties are taken into account in the parameters:\n",
+    "\n",
+    "- the initial U.S. population $y_0$,\n",
+    "- the parameters $a$ and $b$.\n",
+    "\n",
+    "Indeed, Pearl and Reed [2] estimated the parameters $a$ and $b$ using the U.S. census data from 1790 to 1910 while we have the data up to 2000. Moreover, the method used by Pearl and Reed to estimate the parameters could be improved; they only used 3 dates to estimate the parameters instead of using least squares, for example. Finally, Pearl and Reed did not provide confidence intervals for the parameters $a$ and $b$. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Normalizing the data\n",
+    "\n",
+    "The order of magnitude of the population is $10^6$. This is why we consider the normalized population in millions:\n",
+    "\n",
+    "$$\n",
+    "z(t) = \\frac{y(t)}{10^6}\n",
+    "$$\n",
+    "\n",
+    "for any $t\\in[t_0, t_{final}]$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Uncertainties\n",
+    "\n",
+    "We consider that the parameters $z_0$, $a$, $b$ are independent random variables with the following gaussian distributions.\n",
```

Comment:
```suggestion
    "Uncertainty can be accounted for by turning $z_0$, $a$ and $b$ into independent random variables $Z_0$, $A$ and $B$ with Gaussian distributions. From this point onward, $z_0$, $a$ and $b$ respectively denote $\mathbb{E}[Z_0]$, $\mathbb{E}[A]$ and $\mathbb{E}[B]$.\n",
```

Reply:
I used the same content as in calibration_logistic.ipynb. However, in the long run, we should find a strategy to separate the general description of the physical use-case and the content of a specific example. I have no idea yet. Fixed.